### PR TITLE
Add cmake-format to dev-requirements.txt

### PR DESCRIPTION
**Pull request by joakim-hove**

### CODE REVIEW

**Comment by oyvindeide** [comment]
@sondreso can you check why snyk failed?

File: dev-requirements.txt [hunk 1]
```diff
@@ -18,4 +18,6 @@ click
 scikit-build
 setuptools_scm
 pytest-snapshot
-flake8>=4.0.0
\ No newline at end of file
+flake8>=4.0.0
+cmake-format
```
Comment (dafeda) [inline dev-requirements.txt:22]: Suggest we add the lower bound `>= 0.6.0`.

**Review by dafeda — APPROVED** [review]

**Comment by sondreso** [comment]
image
But this is fine, so when this is approved I will use admin powers to get in in :slightly_smiling_face:

File: dev-requirements.txt (hunk 1 above)
Comment (joakim-hove) [reply dev-requirements.txt:22]: Ehh - I was about to address this, but then it was merged. I'll leave it.

Comment (dafeda) [reply dev-requirements.txt:22]: No worries, just me being overly pedantic :)

**Pull request merged**